resources/elasticache-memcacheclusters.go

```diff
@@ -62,21 +62,38 @@ func (l *ElasticacheCacheClusterLister) List(_ context.Context, o interface{}) (
 		}
 
 		resources = append(resources, &ElasticacheCacheCluster{
-			svc:       svc,
-			clusterID: cacheCluster.CacheClusterId,
-			status:    cacheCluster.CacheClusterStatus,
-			Tags:      tags.TagList,
+			svc:        svc,
+			clusterID:  cacheCluster.CacheClusterId,
```

Comment:
```suggestion
			ClusterID:  cacheCluster.CacheClusterId,
```

Reason: libnuke now has this automatic properties generator based on the resource struct.